code/modules/mob/living/carbon/human/species/station/station_special_abilities_ch.dm

```diff
@@ -0,0 +1,160 @@
+//succuby bite is back baby
```

Comment:
Enable this file in the .dme. Add a define for it so they actually get called in neutral traits

Reply:
updated